src/Elmish.WPF/Binding.fs

```diff
@@ -21,7 +21,7 @@ type internal OneWaySeqLazyData<'model, 'a, 'b, 'id when 'id : equality> =
     Equals: 'a -> 'a -> bool
     GetId: 'b -> 'id
     ItemEquals: 'b -> 'b -> bool }
-    
```

Comment:
Sorry, I will revert all of these whitespace changes (I forgot again to disable the VS extension removing them on save).

Reply:
I prefer to have no trailing spaces at the end of a line, so your PR removes some of them, then feel free to leave them in (as long as the diff is easy to understand).

Reply:
I will do a separate PR with trailing space removal.